src/bundle/ui-dev/src/modules/content-tree/components/list-item/list.item.component.js

```diff
@@ -19,6 +20,15 @@ class ListItem extends Component {
         };
     }
 
+    getPrefixActions() {
+        const { prefixActions } = window.eZ.adminUiConfig.contentTreeWidget;
+        const prefixActionsArr = prefixActions ? [...prefixActions] : [];
+
+        return prefixActionsArr.sort((prefixActionA, prefixActionB) => {
```

Comment:
Maybe this would be clearer?
```suggestion
        if (!prefixActions) {
            return [];
        }

        return [...prefixActions].sort((prefixActionA, prefixActionB) => {
```